samples/matter/lock/configuration/nrf7002dk_nrf7001_nrf5340_cpuapp/pm_static_dfu.yml

```diff
@@ -0,0 +1,56 @@
+mcuboot:
```

Comment:
This clashes with #14281 but I'll let you and @ArekBalysNordic decide which PR goes first :)

Reply:
@ArekBalysNordic WDYT? Shall I go ahead with this merge?

Reply:
We changed the layout recently. Please rename it to `pm_static_nrf7002dk_nrf7001_nrf5340_cpuapp.yml` and move it to the main sample's directory. There is no "configuration" directory within the sample anymore on NCS main.

Reply:
Do we want to support the "release" version? If so you can create the same file but with the suffix `_release.yml`: `pm_static_nrf7002dk_nrf7001_nrf5340_cpuapp_release.yml` and store it in the main sample's directory.